packages/blade/src/components/ChatBubble/types.ts

```diff
@@ -0,0 +1,42 @@
+import type React from 'react';
+import type { StyledPropsBlade } from '~components/Box/styledProps';
+import type { DataAnalyticsAttribute, StringChildrenType, TestID } from '~utils/types';
+
+type CommonChatBubbleProps = {
+  isLoading?: boolean;
+  validationState?: 'error' | 'none';
+  errorText?: string;
+  onClick?: () => void;
+  footerActions?: React.ReactNode;
+  children?: React.ReactNode | StringChildrenType;
+} & TestID &
+  StyledPropsBlade &
+  DataAnalyticsAttribute;
+
+type DefaultMessageBubbleProps = {
+  children: React.ReactNode | string;
+  leading?: React.ReactNode;
+  isLoading?: boolean;
+  onClick?: () => void;
+};
+
+type SenderChatBubbleProps = CommonChatBubbleProps & {
+  senderType: 'self';
+  messageType: 'default' | 'last';
```

Comment:
Add jsdoc comments for inline documentation in these props. 

Like what is the default value of messageType, how to use footerActions etc, see other components to get better idea.